concourse/pipeline.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       - task: publish
         file: fauna-vscode-repository/concourse/tasks/publish.yml
         params:
-          VSCE_PAT: ((vscode-marketplace-token_expires-2023_06_29))
+          VSCE_PAT: ((vscode-marketplace-token_expires-2024_09_09))
```

Comment:
Did you mean to add this line in this change?

Reply:
yes, we need to use the new PAT, the current one the pipeline uses to publish is expired